try:
    assert False, "This is an assertion error"
except Exception as e:
    print("Message:", e)
